### DISCUSSION TITLE
Firebase + Recaptcha + Google cloud Identity

### DISCUSSION THREAD

**Discussion by fnarbona** [invertase/react-native-firebase#6201] [General]
I am trying to set up MFA correctly, and your docs suggests using Google Cloud Identity [here](https://rnfirebase.io/auth/phone-auth#mfa-like-account-creation), but the guide for google cloud identity MFA sets you up for web. I don't see the ability to call RecaptchaVerifier from your library's auth instance. Please advise...

**Comment by mikehardy** [comment]
Multi-factor auth with phone flow has been implemented released in v11.3.0 here
